### DISCUSSION TITLE
IP address and port clarification.

### DISCUSSION THREAD

**Discussion by karabomatabane** [caobo171/node-zklib#45] [Q&A]
Hi. My networking knowledge is a bit limited and I'm not sure where the included IP and ports come from. Do I find them in the settings of my ZKTeco device? Please explain it to me like I'm five ^...^

**Comment by ryunosuke21** [comment]
What I managed to find is that the number between the '000' is the IP address, the second is the port (Usually 4370), third is the connection timeout (How long it will try to open a socket) and the last for how long it will try to request the data. So in this case its trying to go to the device '10.20.0.6' on port 4370 for 10000 ms, if it reaches it, then the data gets pulled in for about 4000ms.

**Comment by karabomatabane** [comment]
Thank you for the clarification :)
